etc/systemd/system/zfs-import-cache.service.in

```diff
@@ -5,10 +5,11 @@ Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=cryptsetup.target
 Before=dracut-mount.service
-ConditionPathExists=@sysconfdir@/zfs/zpool.cache
+ConditionPathExists=${ZPOOL_CACHE}
```

Comment:
We should make this `${ZPOOL_CACHE:-@sysconfdir@/zfs/zpool.cache}` to support the case where there is no EnvironmentFile.